postgres-docker/update_common.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+curl -o common.sh https://raw.githubusercontent.com/sclorg/postgresql-container/master/9.4/root/usr/share/container-scripts/postgresql/common.sh && patch common.sh < multiple_dbs.patch
```

Comment:
One comment to: `patch common.sh < multiple_dbs.patch`

I think you don't need to specify which file you want to patch since the patch file already contains the path.
`patch < multiple_dbs.patch` seems to be working as well.

Reply:
good catch, updated in the latest commit.